File: Java/Day_1/Factorial.java
import java.util.Scanner;

class Factorial {

    public int factorialCal(int n) {

        int mul = 1;

        for (int i = 1; i <= n; i++) {
            mul *= i;
        }

        return mul;
    }

    public static void main(String[] args) {
        System.out.println("Enter the number: ");
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();

        Factorial fact = new Factorial();

        System.out.println("Factorial of " + num + " is " + fact.factorialCal(num));
        // System.out.println("Factorial of " + num + " is " + factorialCal(num));
        // System.out.println("Factorial of " + num + " is " + Factorial.factorialCal(num));

        sc.close();
    }
}

File: C++/Code/c++/Day 3/Factorial.cpp
#include <iostream>
using namespace std;

int main()
{
    int num, flag = 0;

    cout<<"Enter the number = ";
    cin>>num;
    
    for (int i = 2; i < num; i++)
    {
        if (num%i == 0)
        {
            flag++;
            // break;
        }   
    }

    if (flag > 0)
    {
        cout<<"The number is not prime";
    }
    else
    {
        cout<<"The number is prime";
    }
    
}